app/models/policies/international_relocation_payments/eligibility.rb

```diff
@@ -3,7 +3,14 @@ module InternationalRelocationPayments
     class Eligibility < ApplicationRecord
       self.table_name = "international_relocation_payments_eligibilities"
 
+      AMENDABLE_ATTRIBUTES = %i[].freeze
+
       has_one :claim, as: :eligibility, inverse_of: :eligibility
+      belongs_to :current_school, class_name: "School", optional: true
```

Comment:
Does the current IRP app set a school? If not, could this just be a method like

```
# required for compatibility with ...
def current_school
  nil
end
```

Reply:
Actually I'm not sure what we want to do here!
There is a school address page `step/employment-details` 🤔

Reply:
I think for the minute we can just have `current_school` as a method that returns nil and we'll just capture the school name / address as is currently done in IRP. Easy enough to change if we do need an association